src/main/java/core/basesyntax/model/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.model;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started.");
```

Comment:
The message should be more informative and indicate that it is a Bulldozer that is working. For example: "Bulldozer is doing its work."